functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -108,7 +109,103 @@ var _ = Describe("[rfe_id:OCP-38968][ppc] Performance Profile Creator", func() {
+			Expect(ppcPath).To(BeAnExistingFile())
+			mustGatherFullPath := path.Join(mustGatherPath, "must-gather.bare-metal")
+			Expect(mustGatherFullPath).To(BeADirectory())
+			cmdArgs := []string{
+				fmt.Sprintf("--disable-ht=%t", false),
+				fmt.Sprintf("--mcp-name=%s", "worker-cnf"),
+				fmt.Sprintf("--must-gather-dir-path=%s", mustGatherFullPath),
+				fmt.Sprintf("--reserved-cpu-count=%d", 100),
+				fmt.Sprintf("--rt-kernel=%t", true),
+				fmt.Sprintf("--user-level-networking=%t", false),
+				fmt.Sprintf("--profile-name=%s", "Performance"),
+			}
+			_, err := testutils.ExecAndLogCommand(ppcPath, cmdArgs...)
+			ppcErrorString := errorStringParser(err)
+			Expect(ppcErrorString).To(ContainSubstring("failed to compute the reserved and isolated CPUs: please specify the reserved CPU count in the range [1,79]"))
+		})
+
+		It("[test_id:41421] Verify PPC script fails when odd number of reserved cpus are specified", func() {
+			Expect(ppcPath).To(BeAnExistingFile())
+			mustGatherFullPath := path.Join(mustGatherPath, "must-gather.bare-metal")
+			Expect(mustGatherFullPath).To(BeADirectory())
+			cmdArgs := []string{
+				fmt.Sprintf("--disable-ht=%t", false),
+				fmt.Sprintf("--mcp-name=%s", "worker-cnf"),
+				fmt.Sprintf("--must-gather-dir-path=%s", mustGatherFullPath),
+				fmt.Sprintf("--reserved-cpu-count=%d", 5),
+				fmt.Sprintf("--rt-kernel=%t", true),
+				fmt.Sprintf("--user-level-networking=%t", false),
+				fmt.Sprintf("--profile-name=%s", "Performance"),
+			}
+			_, err := testutils.ExecAndLogCommand(ppcPath, cmdArgs...)
+			ppcErrorString := errorStringParser(err)
+			Expect(ppcErrorString).To(ContainSubstring("failed to compute the reserved and isolated CPUs: can't allocate odd number of CPUs from a NUMA Node"))
+		})
+	})
+	Context("Systems with Hyperthreading disabled", func() {
+		It("[test_id:42035] verify PPC script fails when splitting of reserved cpus and single numa-node policy is specified", func() {
+			Expect(ppcPath).To(BeAnExistingFile())
+			mustGatherFullPath := path.Join(mustGatherPath, "must-gather.bare-metal")
+			Expect(mustGatherFullPath).To(BeADirectory())
+			cmdArgs := []string{
+				fmt.Sprintf("--disable-ht=%t", true),
+				fmt.Sprintf("--mcp-name=%s", "worker-cnf"),
+				fmt.Sprintf("--must-gather-dir-path=%s", mustGatherFullPath),
+				fmt.Sprintf("--reserved-cpu-count=%d", 2),
+				fmt.Sprintf("--rt-kernel=%t", true),
+				fmt.Sprintf("--split-reserved-cpus-across-numa=%t", true),
+				fmt.Sprintf("--user-level-networking=%t", false),
+				fmt.Sprintf("--profile-name=%s", "Performance"),
+				fmt.Sprintf("--topology-manager-policy=%s", "single-numa-node"),
+			}
+			_, err := testutils.ExecAndLogCommand(ppcPath, cmdArgs...)
+			ppcErrorString := errorStringParser(err)
+			Expect(ppcErrorString).To(ContainSubstring("failed to obtain data from flags not appropriate to split reserved CPUs in case of topology-manager-policy: single-numa-node"))
```

Comment:
same, I'd just check for
"not appropriate to split reserved CPUs in case of topology-manager-policy: single-numa-node"